docs/software/media.md

```diff
@@ -111,7 +112,7 @@ Also installs:
 
 - MPD (music player daemon)
 
-![myMPD web interface screenshot](../assets/images/dietpi-software-media-mympd.gif){: width="400" height="318" loading="lazy"}
+![myMPD web interface screenshot](../assets/images/dietpi-software-media-mympd.png){: width="500" height="287" loading="lazy"}
```

Comment:
If the size is set to 500x287 px² here, then the image shouldn't be 1403x645 px² to save bandwidth etc.

Reply:
Image sizes are optimized - together with further images - in another PR (#813).